pyproject.toml

```diff
@@ -16,12 +16,13 @@ requests = "^2.23.0"
 ipdb = "^0.13.2"
 gunicorn = "^20.0.4"
 coverage = "^5.1"
+flask-socketio = "^4.3.0"
+eventlet = "^0.25.2"
```

Comment:
using the `flask-socketio` library and the `eventlet` service to implement the websockets